src/workers/cdc.ts

```diff
@@ -150,13 +150,18 @@ const onUserChange = async (
   }
 };
 
+const anyHasChanged = (
+  after: Record<string, unknown>,
+  before: Record<string, unknown> = {},
+) => Object.entries(after).some(([key, value]) => before[key] !== value);
+
 const onAlertsChange = async (
   con: Connection,
   logger: Logger,
   data: ChangeMessage<Alerts>,
 ): Promise<void> => {
   if (data.payload.op === 'u') {
-    if (data.payload.before?.filter !== data.payload.after.filter) {
+    if (anyHasChanged(data.payload.after, data.payload.before)) {
```

Comment:
If this is an update operation, it means that something has changed so this is pretty redundant

Reply:
I was thinking to save some resources since when any column has triggered an update even the new value is the same as the value before, this part will still be triggered. Just to safeguard the usage since our update mechanism for this table is very general but we can definitely remove this comparison since updates from this table are not very frequent.

Reply:
An update event will not be triggered if the values are the same even if there was an update statement. The safeguard is not needed here :)

Reply:
Ah, I see. Thought it would run the same as how it would on local test cases where it did execute.

Though if that's the case, then we should just remove this extra safeguard check.